.github/workflows/ci.yml

```diff
@@ -62,9 +62,17 @@ jobs:
         run: |
           sudo apt-get install pandoc
           pip install -r requirements/docs.txt
+          pip install -e .
```

Comment:
Is it necessary to install `alibi` and all its runtime dependencies here? We need to keep in mind also that on `readthedocs` the available dependencies are limited to `requirements/docs.txt` + `pandoc`.

Reply:
Thanks, removed this.